pkg/capabilities/capabilities.go

```diff
@@ -546,4 +580,7 @@ type CapabilityConfiguration struct {
 	RemoteTriggerConfig    *RemoteTriggerConfig
 	RemoteTargetConfig     *RemoteTargetConfig
 	RemoteExecutableConfig *RemoteExecutableConfig
+
+	// v2 / "NoDAG" capabilities
+	CapabilityMethodConfig map[string]CapabilityMethodConfig
```

Comment:
I'll add a comment here saying the key has to match the exact literal value of the capability's [proto file](https://github.com/smartcontractkit/chainlink-protos/blob/main/cre/capabilities/blockchain/evm/v1alpha/client.proto#L259C7-L259C17) like `LogTrigger` (trigger), `WriteReport` (action/executable), `GetTransactionByHash` (action/executable) as the generated Go code for the actions will be consistent, but for the trigger will prepend the `Register`, such as `RegisterLogTrigger` and the method to be added to that map is `LogTrigger` and not `RegisterLogTrigger`)